### PR TITLE
change of theme to Material and some syntax updates

### PR DESCRIPTION
**Pull request by 0kashi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 site/*
+.DS_Store
```
Comment (wgordon17) [inline .gitignore:2]: 🤣 Good call

**Review by wgordon17 — APPROVED** [review]
LGTM

File: OSD4/1-prereqs.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ You will need to [download and install](https://docs.openshift.com/dedicated/cli
 
 ![CLI Tools](images/0-cli_tools_page.png)
 
-##### Why use `oc` over `kubectl`
+**Why use `oc` over `kubectl`**<br>
```
Comment (wgordon17) [inline OSD4/1-prereqs.md:10]: Is the `<br>` valid here?

Comment (0kashi) [reply OSD4/1-prereqs.md:10]: I added that because for some reason it was all appearing on the same line in local testing.

**Pull request merged**